Makefile

```diff
@@ -13,58 +13,26 @@
 # limitations under the License.
 #
 
-# Set by cloudbuild (see cloudbuild.yaml)
-# TAG will contain a tag of the form vYYYYMMDD-hash, vYYYYMMDD-tag, or vYYYYMMDD-tag-n-ghash,
-# depending on the git tags on your repo. The test-infra docs recommend using $_GIT_TAG
-# to tag our images.
-# PULL_BASE_REF will contain the base ref that was pushed to - for instance, master or
-# release-0.2 for a PR merge, or v0.2 for a tag.
-
 SOURCES := $(shell find . -name '*.go')
 GOOS ?= $(shell go env GOOS)
+GOPROXY ?= $(shell go env GOPROXY)
 GIT_VERSION := $(shell git describe --match=$(git rev-parse --short=8 HEAD) --always --dirty --abbrev=8)
-LDFLAGS := "-w -s -X 'main.version=$(GIT_VERSION)'"
-ifneq ($(GOPROXY),)
-	GOPROXYFLAG := --build-arg GOPROXY=$(GOPROXY)
-endif
-
-# Registry and image name
-STAGING_REGISTRY := gcr.io/k8s-staging-provider-aws
-REGISTRY ?= $(STAGING_REGISTRY)
-IMAGE_NAME ?= cloud-controller-manager
-
-# tags
-ifneq ($(TAG),)
-	DEV_TAG = $(TAG)
-else
-	DEV_TAG = $(GIT_VERSION)
-endif
-IMAGE ?= $(REGISTRY)/$(IMAGE_NAME):$(DEV_TAG)
-
-RELEASE_TAG := $(shell git describe --abbrev=0 2>/dev/null)
-RELEASE_IMAGE ?= $(REGISTRY)/$(IMAGE_NAME):$(RELEASE_TAG)
+VERSION ?= $(GIT_VERSION)
+IMAGE := amazon/cloud-controller-manager:$(VERSION)
```

Comment:
is this a real image or just a dummy replacement?

Reply:
Dummy replacement for now, but would be replaced by something real.

Reply:
Well to be clear, I was really intending this to just be the name we gave it so that it makes sense locally when listed with `docker images`, and the official image logic for gcr, ecr (when we add it) would be explicit, e.g. the the gcr logic is entirely contained in the cloudbuild.yaml file.